sellotape/sellotape_dj/main_app/views.py

```diff
@@ -1,4 +1,45 @@
 from django.shortcuts import render
+import datetime
+from.models import Stream, User
+# pip install --upgrade google-api-python-client
+from apiclient.discovery import build
+
+api_key = "AIzaSyCAL9HUYTpY9vX05h2Wnk5CAmYmPvSxX8E"
+youtube = build('youtube', 'v3', developerKey=api_key)
 
 def index(request):
     return render(request, 'sellotape/index.html')
+	
+def get_channel_videos(request):
+    #shuold be added to User
+    channel_id = request.user.get_youtube_link()
+    # get Uploads playlist id
+    res = youtube.channels().list(id=channel_id, part='contentDetails').execute()
+    playlist_id = res['items'][0]['contentDetails']['relatedPlaylists']['uploads']
```

Comment:
This seems like you're trying to fetch the  user `uploads` and not live streams and may result in many of his uploads and not previous/scheduled streams that he may have uploaded/scheduled on youtube.

Shouldn't we use the [Live Streaming API](https://developers.google.com/youtube/v3/live/getting-started)?